test-engineering/contract-tests/partner/app/partner_models.py

```diff
@@ -43,22 +40,16 @@ class ResponseFromFile(BaseModel):
     delay: float = 0.0
 
 
-class QueryParameter(BaseModel):
+class QueryParameter(BaseModel, frozen=True):
     """Model that represents a HTTP query parameter."""
 
```

Comment:
If you'd like to maintain the immutability as was defined with `frozen=True`, so instance values cannot be altered once created, you can define it within the class not using `model_config` like below:
 
```suggestion
from pydantic import ConfigDict

class QueryParameter(BaseModel):
    """Model that represents a HTTP query parameter."""
    model_config = ConfigDict(frozen=True)
    
    name: str
    value: str
```